src/RequestProcessor.cs

```diff
@@ -344,6 +344,19 @@ internal StreamingMessage ProcessInvocationCancelRequest(StreamingMessage reques
 
         internal StreamingMessage ProcessFunctionEnvironmentReloadRequest(StreamingMessage request)
         {
+            var stopwatch = new Stopwatch();
+            stopwatch.Start();
+
+            var environmentReloadRequest = request.FunctionEnvironmentReloadRequest;
+            foreach (var (name, value) in environmentReloadRequest.EnvironmentVariables)
+            {
+                Environment.SetEnvironmentVariable(name, value);
```

Comment:
We only want to set the variables for the current process, is that correct?

Reply:
Yes. Each worker will receive its own reload request and will take care of itself only.

Reply:
Thanks for the clarification.